tools/wagmi-example/README.md

```diff
@@ -0,0 +1,25 @@
+# Wagmi Hedera Example
+
+Wagmi is a library for building Web3 apps, offering hooks (React version) for wallet connections, blockchain data, and transactions.
+
+Learn more [here](https://wagmi.sh).
+
+## How to run example?
+1. Install dependencies `npm install`
+2. Run project `npm dev`
+
+## Example
+Example of using Wagmi with Hedera covers
+1. Sign in with wallet
+2. Check account balance
+3. Deploy contract from bytecode
+4. Read from contract
+5. Write to contract
+
+## Generate contract abi, bin
+If you want to compile the contract on your own, be aware of a known issue with the two latest versions of the Solc compiler. You should use version solc <= 0.8.24 because the most recent versions of Hedera do not support all the opcodes introduced in the latest releases.
```

Comment:
Q: (side note) what opcodes are not supported?
Hedera supports Cancun in general unless there's a concept of blobs.
Those not supported shouldn't block general usage.

Reply:
Comment was removed.